src/Pim/Bundle/EnrichBundle/Resources/config/form_extensions/product_edit.yml

```diff
@@ -14,6 +14,11 @@ extensions:
         aclResourceId: pim_enrich_product_edit_attributes
         position: 100
 
+    pim-product-edit-form-column:
```

Comment:
"Sidebar" is the name of the library used to display the tree, I would use another name. Left column is ok.